sdks/builds/android.mk

```diff
@@ -103,8 +110,16 @@ _android-$(1)_AR=$$(ANDROID_TOOLCHAIN_PREFIX)/$(1)-clang/bin/$(3)-ar
 _android-$(1)_AS=$$(ANDROID_TOOLCHAIN_PREFIX)/$(1)-clang/bin/$(3)-as
 _android-$(1)_CC=$$(CCACHE) $$(ANDROID_TOOLCHAIN_PREFIX)/$(1)-clang/bin/$(3)-clang
 _android-$(1)_CXX=$$(CCACHE) $$(ANDROID_TOOLCHAIN_PREFIX)/$(1)-clang/bin/$(3)-clang++
+ifneq ($(wildcard $$(ANDROID_TOOLCHAIN_PREFIX)/$(1)-clang/bin/$(3)-cpp),)
 _android-$(1)_CPP=$$(CCACHE) $$(ANDROID_TOOLCHAIN_PREFIX)/$(1)-clang/bin/$(3)-cpp
+else
+_android-$(1)_CPP=$$(_android-$(1)_CC) -E
```

Comment:
Can we just use cc -E unconditionally?

Reply:
We probably could, but I wanted to deviate from the way it currently works in most cases (`cpp`) as little as possible - in the spirit of making minimal changes.

Reply:
Minimal changes also lead to larger mess though.

Reply:
True, but they don't disrupt the way things have been working (and tested) so far.